### PR TITLE
Updating job description

### PR DESCRIPTION
**Pull request by msteitle**

### CODE REVIEW

File: content/jobs/javascript-engineer.md [hunk 1]
```diff
@@ -8,23 +8,36 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
 
-In an intermediate role, you would get right to work in a complex application that clients depend on to get their jobs done. You would learn to refine not only your own code but our APIs that it works with. You would have growing responsibility for new features and tested bug fixes. In any given week, you might:
+In an intermediate engineering role, you will immediately begin to add value to a complex application that clients depend on to get their jobs done. You will learn to refine not only your own code but the Crunch APIs that this application is built upon. Your responsibilities will grow to include ownership of new features, the resolution of bugs, and regression-proofing both through comprehensive test coverage. In any given week, you might:
```
Comment (nealrichardson) [inline content/jobs/javascript-engineer.md]: "both through comprehensive test coverage" and something...?

File: content/jobs/javascript-engineer.md [hunk 2]
```diff
@@ -8,23 +8,36 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
 
-* prepare some existing code for an upcoming backend change so nothing breaks
 * pair with senior developers to develop new features
-* puzzle through a client-reported bug; eventually testing then fixing it
+* triage, unravel, test, and resolve client-reported bugs
```
[the remark this answers is not in the document]
Comment (skoszuta) [reply content/jobs/javascript-engineer.md]: `issue tracking` (no s at the end of issue)

File: content/jobs/javascript-engineer.md [hunk 3]
```diff
@@ -8,23 +8,36 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
+Crunch is seeking talented, motivated front-end engineers to build our interactive data applications. As an engineer at Crunch, you will collaborate with others throughout our organization and technology stack to build products that are revolutionizing the way our clients achieve their business goals. As a part of the front-end team, you care greatly about developing solutions that are both technically and aesthetically sound (You know what generator functions and HTML entities are and when/why to use them.) You bring your experience and perspective to conversations about the future growth of our applications and environment. You have a desire to surround yourself with talented peers who share your dedication. In short, you write solid, thoroughly tested, and readable code, and are ready to contribute as part of a highly effective team.
```
[the remark this answers is not in the document]
Comment (skoszuta) [reply content/jobs/javascript-engineer.md]: @kwypchlo that would double the "As a part of the front-end team" expression and also that would make it 3 "As an"'s in a row. I think we should avoid adding another sentence and add those numbers to the existing sentence:
"As a part of the front-end team **(currently four engineers and two designers)**, you care greatly about developing solutions that are both technically and aesthetically sound"

File: content/jobs/javascript-engineer.md [hunk 4]
```diff
@@ -8,23 +8,36 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
+In an intermediate engineering role, you will immediately begin to add value to a complex application that clients depend on to get their jobs done. You will learn to refine not only your own code but the Crunch APIs that this application is built upon. Your responsibilities will grow to include ownership of new features, the resolution of bugs, and regression-proofing both through comprehensive test coverage. In any given week, you might:
 
-* prepare some existing code for an upcoming backend change so nothing breaks
 * pair with senior developers to develop new features
-* puzzle through a client-reported bug; eventually testing then fixing it
+* triage, unravel, test, and resolve client-reported bugs
+* update existing code in parallel with backend changes to ensure uninterrupted operation
+* review peer code contributions to the codebase
+* discuss and debate changes to technology, code organization, design patterns, and conventions employed in the front-end development environment
```
[the remark this answers is not in the document]
Comment (skoszuta) [reply content/jobs/javascript-engineer.md]: We use American English on our website i think: `organization`

File: content/jobs/javascript-engineer.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kwypchlo) [reply content/jobs/javascript-engineer.md]: Sure, it's more about what I would want to read in the job description (what's the team size, are people I will work with juniors or experienced, what is the team structure etc)

**Pull request merged**